### DISCUSSION TITLE
Is there a way to run Autogen Studio 2 with SSL enabled

### DISCUSSION THREAD

**Discussion by dominiclachance** [microsoft/autogen#3213] [Q&A] [answered]
Is there any support for SLL and HTTPS? Has anyone modified it to do so? Thanks for any help.

**Comment by dominiclachance** [comment, marked as answer]
I finally used NGINX.

**Reply by jmaocanada** [reply]
Can you share the steps? Thanks.

**Reply by dominiclachance** [reply]
I installed NGINX in front of the domain and installed the SSL certificate.
Then I configured NGINX as a proxy to the Autogen url and ports.
